R/data_maggie.R

```diff
@@ -168,9 +169,9 @@
 #' library(dplyr)
 #' library(tidyr)
 #' nfl_fandom_surveymonkey_tidy <- nfl_fandom_surveymonkey %>%
-#'   gather(key = race_party, value = percent,
-#'          -c("team", "total_respondents", "gop_percent", "dem_percent",
-#'             "ind_percent", "white_percent", "nonwhite_percent")) %>%
+#'   pivot_longer(-c("team", "total_respondents", "gop_percent", "dem_percent",
```

Comment:
I tried running a local package check of your branch, and I’m getting errors. In particular relating to the `pivot_longer()` code like the one here. Could you fix these? Also, it’s generally good practice to run one final local check before requesting a review

Reply:
fixed!